apps/system/utils/utils_heapinfo.c

```diff
@@ -44,7 +44,7 @@ static char *ptr = CONFIG_HEAPINFO_USER_GROUP_LIST;
 const static char *end_list = CONFIG_HEAPINFO_USER_GROUP_LIST + sizeof(CONFIG_HEAPINFO_USER_GROUP_LIST) - 1;
 #endif
 
-#define HEAPINFO_BUFLEN 64
+#define HEAPINFO_BUFLEN (64 + CONFIG_TASK_NAME_SIZE)
```

Comment:
LGTM.

Reply:
okay, I dig into the final code line.
fs/procfs/fs_procfsproc.c, 
procfile->line[64];
    linesize = snprintf(procfile->line, STATUS_LINELEN, "%d %d %d %d %d %d %d %d %d", tcb->pid,  ppid, tcb->sched_priority, tcb->flags, tcb-     
>task_state, tcb->adj_stack_size, peak_stack, curr_heap, peak_heap);
  item orig type    max size as string(bytes)
tcb->pid:  int16_t,  10bytes
ppid: int16_t,      5bytes  
tcb->sched_priority: uint8_t   3bytes
tcb->flags: uint16_t     5bytes
tcb- >task_state: uint8_t  3bytes 
tcb->adj_stack_size:  uint32_t    10bytes
peak_stack: int  10bytes
curr_heap: int   10bytes 
peak_heap: int  10bytes
so sum up to 66 bytes, consider most cases will not use max value, 64byte is enough.

#if CONFIG_TASK_NAME_SIZE > 0
  linesize = snprintf(procfile->line, STATUS_LINELEN, " %s", tcb->name);
#endif